internal/version/version.go

```diff
@@ -35,6 +36,9 @@ type Info struct {
 	CommitHash string    `json:"commitHash,omitempty"`
 	Version    SemVer    `json:"version"`
 	Release    bool      `json:"release,omitempty"`
+
+	crcOnce       sync.Once
+	commitHashCRC uint32
```

Comment:
Any reason we are not making it part of build flags?

Reply:
uhh, yeah i mean, from a precomputation perspective that will save every process some cycles, but from a correctness perspective it adds an avenue for bugs. this way (don't let it be configured) means we're much more unlikely to get it wrong, and i think correctness here is more important than the perf gain we get from basically denormalizing and repeating the data